index.js

```diff
@@ -0,0 +1,75 @@
+const express = require('express');
+const server = express()
+const cors = require('cors')
+const db = require('./database/dbHelpers.js')
+const bcrypt = require('bcryptjs');
+const session = require('express-session');
+const middleWare = require('./middleware.js')
```

Comment:
Nice job bringing in your protected route middleware. If you end up extending you middleware you could shift to destructuring that object 
```javascript
const { protected, middleware1, middleware2, etc } = require('./middleware.js')
```
then when you pass your middleware to the endpoints it will look like 
```javascript
server.get('/api/users', protect, (req, res) => {})
```